src/layouts/Footer/Footer.spec.jsx

```diff
@@ -24,19 +24,6 @@ describe('Footer', () => {
     config.CODECOV_VERSION = versionNumber
   }
 
-  describe('rendering the feedback link', () => {
```

Comment:
The new designs at [here](https://www.figma.com/design/kbJfmcDgAuptsJR5KB28O9/GH-293-(nav%2C-header%2C-and-related-elements)?node-id=322-1385&t=Y95xMkht0CHlNeYK-0) remove this capability